misaligned.c

```diff
@@ -8,6 +8,10 @@ int printColorMap() {
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
             printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            assert((((i*5 + j)== 1) || ((i*5 + j)== 6)|| ((i*5 + j)== 11)|| ((i*5 + j)== 16)|| ((i*5 + j)== 21)) && (minorColor[i] != minorColor[1]));

```

Comment:
Having asserts in production cod might lead to unwanted termination if not handled carefully.
This is test code and needs to be outside production code.